### PR TITLE
Add styled verify now banner

### PR DESCRIPTION
**Pull request by hallieswan**
https://sagebionetworks.jira.com/browse/DHP-234

### CODE REVIEW

File: src/components/studies/StudyList.tsx [hunk 1]
```diff
@@ -176,6 +178,59 @@ const StudySublist: FunctionComponent<StudySublistProps> = ({
   )
 }
 
+const VerifyBanner: FunctionComponent<{isVerified: boolean | undefined}> = ({isVerified}) => {
+  const [displayVerifyBanner, setDisplayVerifyBanner] = React.useState<boolean>(!isVerified)
+
+  if (typeof isVerified === undefined) {
```
Comment (conundrumgirl) [inline src/components/studies/StudyList.tsx:184]: You can display this conditionally in a parent (below) only if verified !== true. And then you don't need 'isVerified' param

File: src/components/studies/StudyList.tsx [hunk 2]
```diff
@@ -482,6 +533,7 @@ const StudyList: FunctionComponent<StudyListProps> = () => {
           </Container>
         </Box>
       </Box>
+      <VerifyBanner isVerified={isVerified} />
```
Comment (conundrumgirl) [inline src/components/studies/StudyList.tsx:536]: `{isVerified !== true && <VerifyBanner /> }`

File: src/components/widgets/AlertBanner.tsx [hunk 3]
```diff
@@ -83,14 +87,21 @@ const AlertBanner: React.FunctionComponent<AlertBannerProps> = ({
       onClose={() => onClose()}
       severity="error"
       style={{color: textColor, backgroundColor: backgroundColor}}
+      sx={{
```
Comment (conundrumgirl) [inline src/components/widgets/AlertBanner.tsx:90]: nice!

**Review by conundrumgirl — CHANGES_REQUESTED** [review]
a couple of small things

File: src/components/studies/StudyList.tsx (hunk 1 above)
Comment (hallieswan) [reply src/components/studies/StudyList.tsx]: Oh got it, makes sense! Done in next commit

**Review by conundrumgirl — APPROVED** [review]

**Pull request merged**